Userland/Applications/Browser/main.cpp

```diff
@@ -94,6 +95,10 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         return 1;
     }
 
+    TRY(Core::System::pledge("sigaction stdio recvfd sendfd unix fattr cpath rpath wpath proc exec"));
+
+    WebView::ProcessManager::initialize();
```

Comment:
Organizationally, a little weird that the first commit does this init for Qt and AppKit but not Serenity